earth2studio/models/px/fengwu.py

```diff
@@ -349,7 +368,7 @@ def __call__(
                 handshake_dim(coords, key, i)
```

Comment:
Move up into output coords
If you do this then `self.output_coords(coords)` should be called right away prior to forward pass to trigger coord errors before model attempts to execute.

I.e.
```
output_coords = self.output_coords(coords)
return self._forward(x, self.ort), output_coords

Reply:
Done